app/components/utils/system-alert-form.js

```diff
@@ -10,6 +10,16 @@ export default class SystemAlertFormComponent extends Component {
     return this.args.alert;
   }
 
+  get beginDate() {
```

Comment:
@MikiDi Dit was de aanpassing rond `@enableTime` waar ik het begin deze maand over had.

Binnen https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/395bb28a40/app/components/web-components/vl-datepicker.js is er functionaliteit voorzien om de legacy VL-datepicker met moment-data om te laten gaan. Nu, dit leek me wat overbodig met dat dit maar op één locatie nodig is. Mijn voorstel moet je maar eens bekijken.

Op zich vond ik het wel raar dat er initieel een moment-datum wordt doorgegeven. Na het systeembericht op slaan en bericht weer te opnieuw te openen is dit dan weer een gewone datum. Nogal verwarrend dacht ik. Is dat de bedoeling?

Reply:
Lijkt me ok voor nu @brenner-company . Die system-alerts hebben al heel lang geen aandacht gekregen wegens geen prio, maar daar kan in gekuist worden.